core/src/mindustry/world/blocks/logic/MemoryBlock.java

```diff
@@ -1,17 +1,24 @@
 package mindustry.world.blocks.logic;
 
+import arc.scene.ui.layout.Table;
+import arc.util.Align;
 import arc.util.io.*;
 import mindustry.gen.*;
+import mindustry.ui.Styles;
+import mindustry.ui.dialogs.BaseDialog;
 import mindustry.world.*;
 import mindustry.world.meta.*;
 
+import java.awt.*;
```

Comment:
Don't use [incompatible Java features](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md#do-not-use-incompatible-java-features-javautilfunction-javaawt).